aiohttp/server.py

```diff
@@ -388,3 +391,19 @@ def handle_request(self, message, payload):
         self.log_access(message, None, response, self._loop.time() - now)
 
         return drain
+
+    @property
+    def tcp_nodelay(self):
+        return self._tcp_nodelay
+
+    def set_tcp_nodelay(self, nodelay):
+        if nodelay == self._tcp_nodelay:
+            return
+        self._tcp_nodelay = nodelay
+        if self.transport is None:
+            # transport is closed
+            return
+        sock = self.transport.get_extra_info('socket')
```

Comment:
Also it will be nice to cache socket reference in order to speedup setting and remoning nodelay